waspc/cli/src/Wasp/Cli/Command/Telemetry/User.hs

```diff
@@ -30,3 +39,7 @@ readOrCreateUserSignatureFile telemetryCacheDirPath = do
 
 getUserSignatureFilePath :: Path' Abs (Dir TelemetryCacheDir) -> Path' Abs File'
 getUserSignatureFilePath telemetryCacheDir = telemetryCacheDir SP.</> [relfile|signature|]
+
+getUserSignatureFromEnv :: IO (Maybe UserSignature)
+getUserSignatureFromEnv =
+  (fmap . fmap) (UserSignature . hexToString . checksumFromString) (ENV.lookupEnv "WASP_TELEMETRY_USER_ID")
```

Comment:
`(fmap . fmap)` love it. Getting inside that Maybe but preserving the context 🥳

Reply:
Ha yes, it is like a `fmap` but going two levels deep! And you can compose as many fmaps as you want this way, if you have multiple functors one in each other.